lib/templates/deploy.batch.aspnet.core.msbuild16.template

```diff
@@ -0,0 +1,15 @@
+::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::
+:: Deployment
+:: ----------
+
+echo Handling ASP.NET Core Web Application deployment with MSBuild16.
+
+:: 1. Restore, Build and publish
+call :ExecuteCmd "%MSBUILD_16_DIR%\MSBuild.exe" /restore "%DEPLOYMENT_SOURCE%\{ProjectPath}" /p:DeployOnBuild=true /p:configuration=Release /p:publishurl="%DEPLOYMENT_TEMP%" %SCM_BUILD_ARGS%
```

Comment:
/restore is modern way of doing restore/publish, allow restore across references

Reply:
use msbuild 16